### ISSUE TITLE
Create operation for projections seems to miss "Checkpoints Enabled" function

### ISSUE THREAD

**Issue by michael-schnell** [EventStore/EventStoreDB-Client-Java#260]
Creating a projection via the UI and HTTP API has the following options:
- Checkpoints Enabled
- Emit Enabled	
- Track Emitted Streams	
- Enabled	

The class [CreateProjectionOptions](https://eventstore.github.io/EventStoreDB-Client-Java/com/eventstore/dbclient/CreateProjectionOptions.html) only has:
- [emitEnabled](https://eventstore.github.io/EventStoreDB-Client-Java/com/eventstore/dbclient/CreateProjectionOptions.html#emitEnabled(boolean)​)
- [trackEmittedStreams](https://eventstore.github.io/EventStoreDB-Client-Java/com/eventstore/dbclient/CreateProjectionOptions.html#trackEmittedStreams(boolean))

The following functions do not exist:
- **Checkpoints Enabled**
- **Enabled** - According to docummentation [enable](https://eventstore.github.io/EventStoreDB-Client-Java/com/eventstore/dbclient/EventStoreDBProjectionManagementClient.html#enable(java.lang.String))​ should be called after creation instead

<sub>[DEV-228](https://linear.app/eventstore/issue/DEV-228/create-operation-for-projections-seems-to-miss-checkpoints-enabled)</sub>

**Comment by w1am** [comment]
Thanks for pointing that out. It will be amended.

**Comment by YoEight** [comment]
It's a similar situation with #259 

TL;DR: it won't be addressed any time soon from a client perspective.

A lengthier explanation can be found in this comment: https://github.com/kurrent-io/KurrentDB-Client-Java/issues/259#issuecomment-2783922398
